### DISCUSSION TITLE
TerraFarm

### DISCUSSION THREAD

**Discussion by Bjoern-Berlin** [scfmod/FS22_TerraFarm#78] [Q&A] [answered]
**EN**
Hello dear scfmod Team.
Your mod is really great.
Is there any way to create a mod for the paintable ground textures? Since I'm playing on a DLC map "Silverrun Forrest", unfortunately there are few and I can't insert any because it's a DLC. On other maps I have inserted 12 textures.
Thanks Björn

**DE**
Hallo liebes scfmod Team.
Eurer mod ist echt klasse.
Gibt es eine Möglichkeit einen Mod zu erstellen für die malbaren Bodentexturen? Da ich auf einer DLC Karte "Silverrun Forrest" spiele, gibt es leider wenige und ich kann auch keine einfügen, da es ein DLC ist. Auf anderen Karten habe ich 12 Texturen eingefügt.
Danke Björn

**Comment by scfmod** [comment, marked as answer]
Unfortunately afaik there isn't any way to add more ground textures to DLC map
